charts/sdfactory/values.yaml

```diff
@@ -32,7 +32,7 @@ image:
     # -- Set the Image Pull Policy
   pullPolicy: Always
     # -- Image tage is defined in chart appVersion.
-  tag: ""
+  tag: "main"
```

Comment:
We don't accept using main tag in the default values.yaml. Please revert this and use the Chart.yaml appVersion property in your template as default.

Reply:
Updated!